source/d3d8to9_base.cpp

```diff
@@ -10,10 +10,11 @@ static const D3DFORMAT AdapterFormats[] = {
 	D3DFMT_X8R8G8B8,
 	D3DFMT_R5G6B5,
 	D3DFMT_X1R5G5B5,
-	D3DFMT_A1R5G5B5,
-	D3DFMT_A2R10G10B10
+	D3DFMT_A1R5G5B5
 };
 
+IDirect3DDevice9 *pCurrentDeviceInterface = nullptr;
```

Comment:
Shouldn't this be a field of the `Direct3D8` class instead? I'm not sure if it is possible to create different D3D8 instances via `Direct3DCreate8`, but if it was, this could cause race conditions otherwise (very unlikely to happen in the wild, but just theoretically).